benchmark/scenarios/test/test_rate_measurer.py

```diff
@@ -0,0 +1,147 @@
+from flocker.testtools import TestCase
```

Comment:
The tests for the rate measurer were duplicated. Now they are run only once and have been moved to its own file